kcidb/test_db.py

```diff
@@ -19,194 +17,201 @@ def test_schemas_main():
+                {
+                    "id": "kernelci:kernelci.org:619c7e1a2e53f8feb4f2efae",
+                    "origin": "kernelci",
+                    "checkout_id": "_:kernelci:5acb9c2a7bc836e9e5172"
+                                   "bbcd2311499c5b4e5f1"
+                },
+                {
+                    "id": "redhat:redhat.org:619c65f9709de72e90f2efd0",
+                    "origin": "redhat",
+                    "checkout_id": "_:redhat:5acb9c2a7bc836e9619c6"
+                                   "5f9709de72e90f2efd0"
+                }
+            ],
+            "tests": [
+                {
+                    "id": "google:google.org:a19di3j5h67f8d9475f26v11",
+                    "build_id": "google:google.org:a1d993c3n4c448b2"
+                                "j0l1hbf1",
+                    "origin": "google",
+                },
+                {
+                    "waived": False,
+                    "start_time": "2021-11-23T03:52:13.660000+00:00",
+                    "path": "baseline.login",
+                    "status": "PASS",
+                    "id": "kernelci:kernelci.org:619c656de1fb4af479f2efaa",
+                    "build_id": "kernelci:kernelci.org:619c64b171"
+                                "2847eccbf2efac",
+                    "origin": "kernelci"
+                },
+                {
+                    "build_id": "redhat:redhat.org:619c65f970"
+                                "9de72e90f2efd0",
+                    "id": "redhat:redhat.org:b9d8be63bc2abca63165"
+                          "de5fd74f0f6d2f0b0d1c",
+                    "origin": "redhat",
+                    "waived": True,
+                    "status": "DONE"
+                },
+            ]
+        }
+    )
+    data = empty_database.dump()
 
-    driver_source = textwrap.dedent(f"""
-        from unittest.mock import patch, Mock
-        client = Mock()
-        client.dump_iter = Mock(return_value=iter({repr((empty, empty))}))
-        with patch("kcidb.db.Client", return_value=client) as \
-                Client:
-            status = function()
-        Client.assert_called_once_with("bigquery:project.dataset")
-        client.dump_iter.assert_called_once()
-        return status
-    """)
-    assert_executes("", *argv, driver_source=driver_source,
-                    stdout_re=re.escape(json.dumps(empty) + "\n" +
-                                        json.dumps(empty) + "\n"))
+    assert_executes("", *argv,
+                    stdout_re=re.escape(json.dumps(data) + "\n"))
 
 
-def test_query_main():
+def test_query_main(empty_database):
     """Check kcidb-db-query works"""
-    driver_source = textwrap.dedent("""
-        from unittest.mock import patch
-        with patch("kcidb.db.Client"):
-            return function()
-    """)
-    argv = ["kcidb.db.query_main", "-d", "bigquery:project.dataset"]
-    assert_executes("", *argv, driver_source=driver_source)
+
+    empty_database.is_initialized()
```

Comment:
This line doesn't seem to have any effect. Please remove, unless you want to take it somewhere further and just forgot to do that.